documentation/modules/ROOT/partials/modules/all-connectors/obs_ref-connector-incremental-snapshot.adoc

```diff
@@ -1,116 +0,0 @@
-[[{context}-adhoc-snapshot]]
-
-= Ad-hoc snapshot
-
```

Comment:
This file is still references by `mysql.adoc` and others to my knowledge in this PR, so why is this file being randomly removed?

Reply:
This is an older version of the file that I replaced when I was working on the incremental snapshots feature and that I overlooked when cleaning up. The content referenced from other files is now in  `con-connector-incremental-snapshot.adoc`.